### PR TITLE
Assets

### PR DESCRIPTION
**Pull request by crabba**
*Issue #, if available:*

*Description of changes:*

`deploy.sh`:

* Add `--verbose` flag
* Add `--public` flag
* Only use `--acl public-read` in `git sync` when public flag true
* Move `make-dist.sh` call to after flag processing to allow passing verbose flag
* Move `mkdocs build` to inside `site()`
* Use absolute path to call `make-dist.sh`
* Remove `--acl public-read` from `s3-sync()`
* Allow bucket names without `s3://` prefix

`add-dist.sh`:

* Add parameters with `--verbose` flag
* Use absolute paths throughout
* Add precautionary path test before deleting old dist
* Verbose flag replaces `v` flag with summary statement in `tar`, `cp`, `pip`, `zip` commands 
* Catch rate limited failed calls to api.github.com

`configure-deploy.sh`
* Add `--clobber` flag
* Only overwrite `.aws/config` on clobber flag or user approval

`.travis.yml`
* Use clobber parameter when calling `configure-deploy.sh`
* Use public and verbose parameters when calling `deploy.sh`

`nextflow/Dockerfile`
* Source nextflow image from public.ecr.aws/seqera-labs/nextflow

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Review by wleepang — COMMENTED** [review]
Would be good to add the `--verbose` flag to `deploy.sh` where it is configured to be called in `.travis.yml`. The verbosity is good for inspecting CI/CD errors.

File: _scripts/deploy.sh [hunk 1]
```diff
@@ -68,20 +74,23 @@ function s3_sync() {
     echo "syncing ..."
     echo "   from: $source"
     echo "     to: $destination"
-    aws s3 sync \
+    cmd="aws s3 sync \
         --profile $ASSET_PROFILE \
         --region $DEPLOY_REGION \
-        --acl public-read \
```
[the remark this answers is not in the document]
Comment (crabba) [reply _scripts/deploy.sh:74]: When deploying within an institution's account, it seems likely that buckets would be non-public and `s3 sync --acl public-read` will fail with Access Denied.  Maybe add a script parameter `--public` to enable public-read?

[the remark this answers is not in the document]
Comment (wleepang) [reply _scripts/deploy.sh:74]: That would work

File: _scripts/deploy.sh [hunk 2]
```diff
@@ -68,20 +97,24 @@ function s3_sync() {
...
+        $ACL_PUBLIC_READ \
         --delete \
         --metadata commit=$(git rev-parse HEAD) \
         $source \
-        $destination
+        $destination"
+    echo $cmd
+    eval $cmd
+    exit
```
Comment (wleepang) [inline _scripts/deploy.sh:110]: is this `exit` necessary?

Comment (crabba) [reply _scripts/deploy.sh]: Left behind in error; removed.

**Comment by crabba** [comment]
deploy.sh resolved cross-platform sed issue in pin_version()
Source nextflow Docker image from public.ecr.aws
Ready for review

**Review by wleepang — APPROVED** [review]

**Pull request merged**